cmd/k3s/main.go

```diff
@@ -207,7 +209,7 @@ func extract(dataDir string) (string, error) {
 		return dir, nil
 	}
 
-	logrus.Infof("Preparing data dir %s", dir)
+	logrus.Debugf("Preparing data dir %s", dir)
```

Comment:
Why are we removing this from normal operation?

Reply:
Reverted